core/controllers/acl_decorators_test.py

```diff
@@ -1632,6 +1634,150 @@ def test_viewer_cannot_accept_suggestion(self):
         self.logout()
 
 
+class DecoratorForUpdatingSuggestionTests(test_utils.GenericTestBase):
+    """Tests for can_update_suggestion decorator."""
+
+    owner_username = 'owner'
+    owner_email = 'owner@example.com'
+    author_username = 'author'
+    author_email = 'author@example.com'
+    reviewer_email = 'reviewer@example.com'
+    username = 'user'
+    user_email = 'user@example.com'
+    TARGET_TYPE = 'exploration'
+    exploration_id = 'exp_id'
+    target_version_id = 1
+    change_dict = {
+        'cmd': 'add_translation',
+        'content_id': 'content',
+        'language_code': 'hi',
+        'content_html': '<p>old content html</p>',
+        'state_name': 'State 1',
+        'translation_html': '<p>Translation for content.</p>'
+    }
+
+    class MockHandler(base.BaseHandler):
+        GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+        @acl_decorators.can_update_suggestions
+        def get(self, suggestion_id):
+            self.render_json({'suggestion_id': suggestion_id})
+
+    def setUp(self):
+        super(DecoratorForUpdatingSuggestionTests, self).setUp()
+        self.signup(self.author_email, self.author_username)
+        self.signup(self.user_email, self.username)
+        self.signup(self.owner_email, self.owner_username)
+        self.signup(self.reviewer_email, 'reviewer1')
+        self.author_id = self.get_user_id_from_email(self.author_email)
+        self.owner_id = self.get_user_id_from_email(self.owner_email)
+        self.reviewer_id = self.get_user_id_from_email(
+            self.reviewer_email)
+        self.owner = user_services.get_user_actions_info(self.owner_id)
+        self.author = user_services.get_user_actions_info(self.author_id)
+        user_services.update_user_role(self.owner_id, feconf.ROLE_ID_ADMIN)
```

Comment:
We should probably call this user an admin rather than an owner. 'Owner' could be confused with someone who 'owns' the suggestion (or other types of structures on Oppia). If this users is an admin, that's what we should designate it as.